### PR TITLE
[feature] listing flow UI

### PR DESCRIPTION
**Pull request by kevinbae15**
- Some plain ui for listing a book and seeing a confirmation page.
- Included an nvmrc so we have a consistent node version
- Clean up

\* Not fully functional. Still have to write to DB.

https://github.com/HongjieWang98/TEN-techteam-coworking/assets/24871169/630e0cda-1546-4867-a92b-eefd866cd02f

### CODE REVIEW

File: src/pages/Sell/ListingPage.jsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    setIsLoading(true);
+
+    if (formRef?.current) {
+      const formData = new FormData(formRef.current);
+      const data = {};
+      formData.forEach((value, key) => {
+        data[key] = value;
+      });
+
+      console.log(data);
```
Comment (Emerson-Wenzel) [inline src/pages/Sell/ListingPage.jsx:22]: Did you run ESLint? I thought console.log should throw an error for console.log

**Review by Emerson-Wenzel — APPROVED** [review]
LGTM

File: src/pages/Sell/ListingPage.jsx (hunk 1 above)
Comment (kevinbae15) [reply src/pages/Sell/ListingPage.jsx:22]: i did not lol, i can remove

Comment (kevinbae15) [reply src/pages/Sell/ListingPage.jsx:22]: will do this and other lint/formatting stuff in another PR

**Pull request merged**